rcgen/src/key_pair.rs

```diff
@@ -126,11 +128,13 @@ impl KeyPair {
 	}
 
 	/// Parses the key pair from the ASCII PEM format
+	///
+	/// The key must be in the PKCS#8 format.
```

Comment:
For these comments, maybe we should mention the `PRIVATE KEY` string used for the PEM delimiters, similar to what we have in rustls-pemfile?